HPHBhelper/hogwarts/views.py

```diff
@@ -6,14 +6,31 @@
 from django.http import HttpResponse
 
 from .forms import GameForm
-from .models import Games
+from .models import Games, Players
 
 
 def lobby(request):
     # Create Game
     # Join Game
     return HttpResponse("Yer not a wizard yet, 'Arry! Welcome to the Lobby. Check back soon.")
 
+def create_player(request):
+    # form = BlogForm(request.POST or None)
+    # if form.is_valid():
+    #     form.save()
+    #     messages.success(request, 'Added post')
+    #     return redirect('')
+    # return render(request, 'blog/form.html', {'form': form})
+    pass
+
+def active_game(request, shortcode):
+    game = Games.object.find(shortcode=shortcode)
+    active_players = Players.object.filter(game=game)
+    if active_players:
+        return render(request, 'game.html', {'shortcode': shortcode, 'active_players': active_players})
+    return render(request, 'game.html', {'shortcode': shortcode})
```

Comment:
repeated code, probably don't need the if, so you can just do `return render(request, 'game.html', {'shortcode': shortcode, 'active_players': active_players})`